client/src/components/AddItemPopup.js

```diff
@@ -50,8 +51,25 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const AddItemPopup = (props) => {
-  const { item, popupOpen, closePopup } = props;
+  const { item, popupOpen, closePopup, listId } = props;
   const classes = useStyles();
+  const listToggle = useContext(ListContext).listToggle;
+
+  const addProductToList = async () => {
+    const res = await fetch(`/list-to-products/${listId}`, {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({ product_id: item.product_id }),
+    });
+    const json = await res.json();
+    return json;
```

Comment:
if we're not doing anything with this return value we don't need to return anything